gateway/src/apicast/policy/logging/readme.md

```diff
@@ -0,0 +1,172 @@
+# Logging policy
+
+This policy has two primary purposes: one is to enable and disable access log
+output, and the second one is to be able to create a custom access log format
+for each service and be able to set conditions to write custom access log. 
+
+
+## Exported variables
+
+Liquid templating can be used on custom logging, the exported variables are:
```

Comment:
`Liquid templating can be used on custom logging. The exported variables include:`